<ui version="4.0" >
 <class>GaussianBeamWindow</class>
 <widget class="QMainWindow" name="GaussianBeamWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GaussianBeam</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Save" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/document-save.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="action_SaveAs" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/document-save-as.png</iconset>
   </property>
   <property name="text" >
    <string>Save as</string>
   </property>
  </action>
  <action name="action_Print" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/document-print.png</iconset>
   </property>
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
  <action name="action_Open" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="action_AddLens" >
   <property name="text" >
    <string>&amp;Lens</string>
   </property>
  </action>
  <action name="action_AddFlatInterface" >
   <property name="text" >
    <string>&amp;Flat Interface</string>
   </property>
  </action>
  <action name="action_AddCurvedInterface" >
   <property name="text" >
    <string>&amp;Cuved Interface</string>
   </property>
  </action>
  <action name="action_AddFlatMirror" >
   <property name="text" >
    <string>Flat &amp;Mirror</string>
   </property>
  </action>
  <action name="action_AddCurvedMirror" >
   <property name="text" >
    <string>Cu&amp;rved Mirror</string>
   </property>
  </action>
  <action name="action_AddGenericABCD" >
   <property name="text" >
    <string>Generic &amp;ABCD</string>
   </property>
  </action>
  <action name="action_AddOptics" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/edit-add.png</iconset>
   </property>
   <property name="text" >
    <string>Add optics</string>
   </property>
  </action>
  <action name="action_RemoveOptics" >
   <property name="icon" >
    <iconset resource="GaussianBeamRessource.qrc" >:/images/edit-delete.png</iconset>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="GaussianBeamRessource.qrc" />
 </resources>
 <connections/>
</ui>
